tests/test_creator/test_creator_get_affiliated_agents.py

```diff
@@ -23,6 +23,17 @@ class School(p2n.MagicLocation):
         for agent in creator._get_affiliated_agents(agents=model.agents, dummy_location=school)
     )
 
+    model = p2n.Model()
+    creator = p2n.Creator(model=model)
+
+    for _ in range(5):
+        agent = p2n.Agent(model=model)
+        agent.gender = "w"
```

Comment:
I still really hate this syntax xD -- this does not convey to me that an agent is added to the model. model.add_agent(agent) is so much clearer :P